mkosi/__init__.py

```diff
@@ -2525,8 +2553,11 @@ def install_debian_or_ubuntu(args: MkosiArgs, root: Path, *, do_run_build_script
         if debootstrap_knows_arg("--no-check-valid-until"):
             cmdline += ["--no-check-valid-until"]
 
-        assert args.mirror is not None
-        cmdline += [args.release, root, args.mirror]
+        if not args.repository_key_check:
+            cmdline += ["--no-check-gpg"]
+
+        assert args.mirror is not None or args.local_mirror is not None
+        cmdline += [args.release, root, typing.cast(Union[Path, str], args.local_mirror if args.local_mirror else args.mirror)]
```

Comment:
```suggestion
        cmdline += [args.release, root, cast(PathString, args.local_mirror or args.mirror)]
```
`cast` is already imported from `typing` and the path string union is used so often that we have this type alias and `a if a else b` can be written as `a or b`.

Reply:
Back in MY day, if a variable walked like a duck and quacked like a duck, then it was a duck! None of this typing malarky!